.github/workflows/build-sanpshot.yml

```diff
@@ -10,6 +10,25 @@ on:
       - README.md
 
 jobs:
+  windows-build-job:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Set up JDK 11 ('zulu')
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - name: Build and test with Gradle
```

Comment:
suggestion: We should add a gradle setup action before we build. In this case, we won't need to download the wrapper every time

```yaml
    - name: Setup Gradle
      uses: gradle/actions/setup-gradle@v4
      with:
        gradle-version: wrapper
```